js/pulse.js

```diff
@@ -6,9 +6,11 @@
   const pulseAdManagers = {};
 
 
-  OO.Ads.manager(function () {
+  OO.Ads.manager((...args) => {
```

Comment:
The same mistake with arguments as in the `liverail.js`. Please fix all appearances